packages/@aws-cdk/aws-appsync/test/appsync-http.test.ts

```diff
@@ -57,6 +57,33 @@ describe('Http Data Source configuration', () => {
     });
   });
 
+  test('appsync configures name, authorizationConfig correctly', () => {
+    // WHEN
+    api.addHttpDataSource('ds', endpoint, {
+      name: 'custom',
+      description: 'custom description',
+      authorizationConfig: {
+        authorizationType: 'AWS_IAM',
```

Comment:
why is this line necessary? the authorizationType is fixed to AWS_IAM no?

Reply:
Yes, It true. AWS_IAM takes only a single value. I remove it.